android/package.json

```diff
@@ -4,11 +4,8 @@
   "readme": "WendriverIO Integration with [BrowserStack](https://www.browserstack.com)",
   "description": "Selenium examples for WebdriverIO and BrowserStack App Automate",
   "scripts": {
-    "test": "npm run first && npm run local && npm run parallel",
-    "first": "./node_modules/.bin/wdio examples/run-first-test/first.conf.js",
-    "parallel": "./node_modules/.bin/wdio examples/run-parallel-test/parallel.conf.js",
-    "local": "./node_modules/.bin/wdio examples/run-local-test/local.conf.js",
-    "multiple": "./node_modules/.bin/wdio examples/run-multiple-test/multiple.conf.js"
+    "test": "./node_modules/.bin/wdio examples/run-parallel-test/parallel.conf.js",
```

Comment:
use npx command, this failed on windows for selenium
for other package.json file as well